ethers-solc/src/config.rs

```diff
@@ -227,7 +228,24 @@ impl ProjectPathsConfig {
             })
         } else {
             // resolve library file
-            self.resolve_library_import(import.as_ref()).ok_or_else(|| {
+            let resolved = self.resolve_library_import(import.as_ref());
+
+            if resolved.is_none() {
+                // absolute paths in solidity are a thing for example `import
+                // "src/interfaces/IConfig.sol"` which could either point to `cwd +
+                // src/interfaces/IConfig.sol`, or make use of a remapping (`src/=....`)
+                // While checking the former first would be safer, it is also less likely and more
+                // expensive, therefor we only check this if resolving the library import resulted
+                // in no path. However, this leaves open an unlikely edge case where a file
+                // `root + import` exists, which is neglected here.
```

Comment:
let's also add a unit test for this edge case?

Reply:
I realized, this is actually not accurate when using `--include-path`